packages/react-microlink/src/components/Card/CardMedia/image.js

```diff
@@ -10,20 +10,39 @@ const largeStyle = css`
   }
 `
 
+const mobileStyle = media.mobile`
+  flex: 0 0 92px;
+`
+
 export default styled.div`
   display: block;
   flex: 0 0 125px;
   background: #e1e8ed no-repeat center center / cover;
+  transition: flex-basis .25s ease-in-out;
+  position: relative;
 
-  &::before {
```

Comment:
why `:` instead of? `::` https://developer.mozilla.org/es/docs/Web/CSS/::before

I always used `::` for pseudo classes 🤔

Reply:
I've always used `::` when entering something in `content`, for some reason. But if it's the css3 standard I'll swap it back 👌